### ISSUE TITLE
No sam init for .net core 2.1

### ISSUE THREAD

**Issue by DamianBis** [awslabs/aws-sam-cli#609]
**Description:**

There is no sam init --runtime for dotnetcore2.1

**Steps to reproduce the issue:**
1. run sam init --help
2. see acceptable --runtime options
 -r, --runtime [go1.x|nodejs|python|python2.7|python3.6|go|nodejs4.3|nodejs6.10|java8|dotnetcore1.0|java|dotnetcore2.0|dotnetcore|dotnet|nodejs8.10]

**Additional environment details (Ex: Windows, Mac, Amazon Linux etc)**
Windows

**Output of `sam --version`**:

SAM CLI, version 0.5.0

**Comment by jfuss** [comment]
@DamianBis Yes, currently we do not have a .NET Core 2.1 sample. Would you be willing to add this? I am unsure if the dontnetcore2.0 will work in dotnetcore2.1 but if it does this should be an easy update.

**Comment by sanathkr** [comment]
@heitorlessa  thoughts?

**Comment by heitorlessa** [comment]
Working on a PR today for that @DamianBis.
